vignettes/introduction-ospsuite.Rmd

```diff
@@ -51,17 +52,40 @@ resultOfAFunction <- object1$multiply(1,2)
 Important information about the object can be printed out by calling `print(object)`.
 
 The most important classes are:
-  : `Simulation` Representation of the simulation loaded from the *.pkml file.
-  : `SimulationSettings` #Short descriptions of the classes to be added here
+  : `Simulation`

```

Comment:
I think an overview of the most important classes with their short descriptions is useful to advanced user.

Reply:
I think it's useful for _all_ users 😄